brokers/vscode/broker.go

```diff
@@ -112,14 +121,22 @@ func (b *Broker) processPlugin(meta model.PluginMeta) error {
 	unpackedPath := filepath.Join(workDir, "plugin")
 
 	// Download an archive
-	b.PrintDebug("Downloading archive '%s' for plugin '%s:%s' to '%s'", url, meta.ID, meta.Version, archivePath)
-	err = b.download(url, archivePath, meta)
-	if err != nil {
-		return err
+	if url != "" {
+		b.PrintDebug("Downloading archive '%s' for plugin '%s:%s' to '%s'", url, meta.ID, meta.Version, archivePath)
```

Comment:
There is a file `./cmd/config.json` which contains example configuration of brokers to process. It is very useful to locally debug a broker. Could you add an example of broker configuration for the case you are adding?

Reply:
Sure

Reply:
thanks!